frappe/core/doctype/data_import/importer_new.py

```diff
@@ -362,14 +365,13 @@ def parse_value(self, value, df):
 		return value
 
 	def parse_date_format(self, value, df):
-		date_format = self.get_date_format_for_df(df)
-		if date_format:
-			try:
-				return datetime.strptime(value, date_format)
-			except:
-				# ignore date values that dont match the format
-				# import will break for these values later
-				pass
+		date_format = self.get_date_format_for_df(df) or '%Y-%m-%d %H:%M:%S'
```

Comment:
should we use the date and time format from system settings instead of hardcoding it? @netchampfaris what do you think?